browser/ui/webui/brave_webui_source.cc

```diff
@@ -167,6 +167,10 @@ void CustomizeWebUIHTMLSource(const std::string &name,
         { "topSitesTitle", IDS_BRAVE_NEW_TAB_TOP_SITES },
         { "statsTitle", IDS_BRAVE_NEW_TAB_STATS },
         { "clockTitle", IDS_BRAVE_NEW_TAB_CLOCK },
+        { "tabClock", IDS_BRAVE_NEW_TAB_CLOCK_TAB },
```

Comment:
I think `clockTitle` (or `xTitle`) was intended for the "tab titles" items, which I think we (I) renamed probably a bit haphazardly. So maybe we should be changing those items to remove the "Show xyz" and adding new items like `clockToggle` which would have the "Show Clock" label?

Hang on what's the `showClock` item that's just above here? If that was from the previous settings dialog and isn't being used anymore, maybe we should be re-using them.